src/components/all.sass

```diff
@@ -1,11 +1,11 @@
 @import "~bulma/sass/utilities/initial-variables"
```

Comment:
Looks like we're using different settings for out Prettier haha?
So many whitespaces

Reply:
I don't have prettier on this file hehe.
But I still think we have different settings since I have single quote on.